floss/language/utils.py

```diff
@@ -403,6 +403,7 @@ def get_struct_string_candidates(pe: pefile.PE) -> Iterable[StructString]:
         if not section.IMAGE_SCN_MEM_READ:
             continue
 
+        # TODO add .text here for Go version 1.12?
```

Comment:
would you add an issue to the tracker so we can be sure not to forget about this and/or solicit help from external contributors?

Reply:
done #872